docs/advisors/checks/mongodb-chunk-imbalance.md

```diff
@@ -0,0 +1,37 @@
+# MongoDB sharding - chunk imbalance across shards
+
+## Description
+This check warns if the chunks are imbalanced across shards.
+
+## Resolution
+
+In a sharded cluster, chunk imbalances can occur when the data is not evenly distributed among the shards. This can lead to some shards having more chunks than others. This can cause performance issues and slower query times.
+
+There are a few possible reasons for chunk imbalance:
+
+- **Poor shard key selection:** If the shard key is not chosen properly, some shards may end up with a larger portion of the data than others. For example, if the shard key is based on a timestamp and the data is inserted in a sequential manner, this can create hotspots due to busier application write periods. 
+
+- **Jumbo chunks:** Jumbo chunks can be another cause of chunk imbalances in a sharded cluster . Jumbo chunks are chunks that have grown beyond the maximum size that is allowed by MongoDB. When a chunk becomes jumbo, MongoDB cannot split it further automatically and the balancer won’t distribute the associated data across the shards. Jumbo chunks are often caused by low cardinality or too high frequency of elements in a shard key.
+
+To address chunk imbalance, you can:
+
+- **Select a good shard key:** 
+The choice of shard key affects the creation and distribution of chunks across the available shards. While selecting the shard key, consider below points:
+
+1. High Cardinality.
```

Comment:
Numbered list shows chronology, imploying there's a procedure that should be followed in the order described. Would it be better to use bullet points instead? 

Also, the list is not parallel and I don't understand the technical details enough to rewrite it. Would you please check these guidelines https://developers.google.com/tech-writing/one/lists-and-tables

Reply:
@catalinaadam We can put bullet points.